api/reader.graphql

```diff
@@ -65,6 +65,10 @@ type Input {
   espressoBlockNumber: String
   "Input index in the Inpux Box"
   inputBoxIndex: String
+  "Type of input. Can be inputbox, Espresso or Avail"
```

Comment:
@DiegoSena we don't want to expose those fields

Reply:
Removed